src/app/components/timeline/TimelineTable.js

```diff
@@ -80,6 +81,23 @@ class TimelineTable extends Component {
     this.props.lineActions.reorderLines(lines, this.props.ui.currentTimeline)
   }
 
+  startHovering = (beat) => {
+<<<<<<< HEAD
```

Comment:
Looks like you missed a conflict here.
Thankfully it looks like a trivial error.